content/en/docs/setup/additional-setup/external-controlplane/index.md

```diff
@@ -247,7 +247,7 @@ Install the configuration on the remote cluster:
 
 {{< text bash >}}
 $ kubectl create namespace external-istiod --context="${CTX_REMOTE_CLUSTER}"
-$ istioctl manifest generate -f remote-config-cluster.yaml  | kubectl apply --context="${CTX_REMOTE_CLUSTER}" -f -
+$ istioctl manifest install -f remote-config-cluster.yaml --context="${CTX_REMOTE_CLUSTER}"
```

Comment:
In 1.9 `istioctl install` will not work in the remote cluster (at least not in a real, non-test, env). That's why the doc explicitly uses `istio manifest generate | kubectl apply`. Has that been fixed recently, where it's now supposed to work?

Also, why do we seem to have two istioctl commands that do the same thing (`istioctl install` and `istioctl manifest install`)?

Reply:
Maybe I should have left the `manifest` out? And thanks for the input on `install` not working for 1.9. That would pretty much mean that I can cancel this PR.